packages/ui-components/src/Tooltip/Tooltip.module.scss

```diff
@@ -90,17 +90,30 @@ $tooltip-arrow-half-width: 5px;
 // Default style
 // --------------------------------------------------
 .default {
-  @include crl-tooltip-style($color-core-neutral-6, $color-core-neutral-6, $color-base-white);
+  @include crl-tooltip-style(
+    $color-core-neutral-6,
+    $color-core-neutral-6,
+    $color-base-white
+  );
+  z-index: 5;
```

Comment:
If you're going to fix this using a `z-index` then we should have some agreement on what `z-index` values we expect to use in the app. Since this field is difficult to use correctly, people put all sorts of numbers into it to "make things work". Can you explain how you got to the number 5?

Reply:
@dhartunian I think the `z-index: 5` is a value that is already used in Tooltips for the arrow elements. But I am also suspicious of the use of z-index here. [Stacking Contexts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context) are notoriously tricky things to deal with in CSS. It is a common misconception that `z-index` is a global or shared value among elements in the DOM, which increases confusion. I haven't read the code yet, but I have to imagine that [`react-popper`](https://popper.js.org/react-popper/v2/) already has some consideration to control the stacking context around the target element.

For the life of me I can't recreate this issue on `master` in Storybook. Are we sure this is still and issue?

Reply:
Issue closed for now, reverted the changes